includes/person-functions.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Person functions
+ */
+
+/**
+ * Handles output for the people list
+ * @author Jim Barnes
+ * @since 1.0.0
+ * @param array $faculty An array of faculty posts
+ * @return string
+ */
+function research_get_faculty_list( $faculty ) {
+	$args = array(
+		'list_title'     => '',
+		'display_search' => false
+	);
+
+	return UCF_Post_List_Common::display_post_list( $faculty, 'people', $args );
```

Comment:
We'll need to make sure that the dependency on the post list sc plugin is documented somewhere.

Reply:
I'll add an issue to make sure this gets added to the wiki along with ACF.